st2common/bin/st2ctl

```diff
@@ -114,15 +114,15 @@ function reopen_component_log_files() {
 }
 
 function register_content() {
-  ALLOWED_REGISTER_FLAGS='--register-all --register-actions --register-aliases --register-policies --register-rules --register-sensors --register-configs --register-setup-virtualenvs --register-fail-on-failure --verbose'
-  DEFAULT_REGISTER_FLAGS='--register-actions --register-aliases --register-sensors --register-configs'
+  ALLOWED_REGISTER_FLAGS='--register-all --register-actions --register-aliases --register-policies --register-rules --register-sensors --register-triggers --register-setup-virtualenvs --register-fail-on-failure --verbose'
+  DEFAULT_REGISTER_FLAGS='--register-actions --register-aliases --register-sensors --register-triggers --register-configs'
```

Comment:
@Kami is this intentional that --register-configs is in DEFAULT but not in ALLOWED? Oversight or intentional?